pkg/asset/templates/kube-controller-manager.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-controller-manager
-    version: v1.3.0-beta.2
```

Comment:
We just can't change label metadata apparently. Update all version labels and then try a `kubectl apply -f` on that updated file and it will refuse to apply the update because of label mismatch.